src/pk_encryption.rs

```diff
@@ -0,0 +1,134 @@
+//! ☣️  Compat support for libolm's PkEncryption and PkDecryption
```

Comment:
I am not sure this will get generated properly in the doc, and I am not sure how I can generate the doc to check.

Reply:
And `pk_encryption` is not registered a Python module. We could do so but unsure about the benefits.

I am thinking about moving this warning to the docstrings of `PkDecryption` and `PkEncryption`.